Tools/Scripts/webkitpy/layout_tests/controllers/layout_test_runner.py

```diff
@@ -499,12 +499,75 @@ def _run_single_test(self, driver, test_input, stop_when_done):
 
 class TestShard(object):
     """A test shard is a named list of TestInputs."""
+    DEFAULT_AVERAGE_TEST_TIME = 256
+    AVERAGE_TEST_TIME_ESTIMATES = {
```

Comment:
Looking at this again, I'm somewhat doubting whether this is the right place to actually store all this data. It's much more like configuration, rather than something tied to the runner?